.gitignore

```diff
@@ -45,6 +45,7 @@
 /config/atc.yml
 /config/aws.yml
 /config/gcp.yml
+/config/gcp-credentials.json
```

Comment:
This ignore is in preparation for an upcoming branch.  Wanted to get it in as early as possible though.

Reply:
I ended up removing this .gitignore line in favor of storing the same info in our existing gcp.yml file (which is already ignored), so we no longer need a `gcp-credentials.json` file.